install/fedora-installation-guide.md

```diff
@@ -3,12 +3,7 @@
 1. Install the Kata Containers components with the following commands:
 
    ```bash
-   $ source /etc/os-release
-   $ ARCH=$(arch)
-   $ BRANCH="${BRANCH:-master}"
-   $ sudo dnf -y install dnf-plugins-core
-   $ sudo -E dnf config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/${BRANCH}/Fedora_${VERSION_ID}/home:katacontainers:releases:${ARCH}:${BRANCH}.repo"
-   $ sudo -E dnf -y install kata-runtime kata-proxy kata-shim
+   $ sudo dnf -y install kata-runtime
```

Comment:
I think that it would be nice to explain that we have both `kata-runtime` that installs 1.x and `kata-containers` that installs 2.x. That would probably avoid confusion about what that `kata-containers` thing is.

Reply:
Hmm.  I'm not sure here.  This repo is specific about 1.x.  Mentioning `kata-containers` and 2.x only for Fedora seems inconsistent.  Changing the whole documentation to mention this about every distro / snap / etc feels like not worth it.

I'd appreciate if I could get the ACK and merge it as it's.